creates.sql

```diff
@@ -0,0 +1,18 @@
+CREATE DATABASE movies;
+
+USE movies;
+
+CREATE TABLE movies(
+id INT AUTO_INCREMENT,
+title VARCHAR(225),
+directors VARCHAR(225), 
+release date VARCHAR(225),
+rating VARCHAR(225),
+run Time VARCHAR(225),
+genres VARCHAR(225),
```

Comment:
Genres should be its own table since there is a many-to-many relationship between genres and movies.